src/main/java/de/teamlapen/vampirism/player/vampire/VampirePlayer.java

```diff
@@ -1212,6 +1226,81 @@ private void updateFeeding() {
             endFeeding(true);
     }
 
+    public static double getNaturalArmorValue(int lvl){
+        return lvl > 0 ? VampirismConfig.BALANCE.vpNaturalArmorBaseValue.get() + (lvl/(double)REFERENCE.HIGHEST_VAMPIRE_LEVEL)*VampirismConfig.BALANCE.vpNaturalArmorIncrease.get():0;
+    }
+
+    public static double getNaturalArmorToughnessValue(int lvl){
+        return  (lvl/(double)REFERENCE.HIGHEST_VAMPIRE_LEVEL)*VampirismConfig.BALANCE.vpNaturalArmorToughnessIncrease.get();
+    }
+
+    public void updateNaturalArmor(){
+        ModifiableAttributeInstance armorAtt = player.getAttribute(Attributes.ARMOR);
+        ModifiableAttributeInstance toughnessAtt = player.getAttribute(Attributes.ARMOR_TOUGHNESS);
+        if(armorAtt!=null&&toughnessAtt!=null){
+
+            int lvl = getLevel();
+            if(lvl==0){
+                armorAtt.removeModifier(NATURAL_ARMOR_UUID);
+                toughnessAtt.removeModifier(NATURAL_ARMOR_TOUGHNESS_UUID);
+            }
+            else{
+                AttributeModifier modArmor = armorAtt.getModifier(NATURAL_ARMOR_UUID);
+                AttributeModifier modToughness = toughnessAtt.getModifier(NATURAL_ARMOR_TOUGHNESS_UUID);
+                double naturalArmor = getNaturalArmorValue(lvl);
+                EffectInstance armorRegen = player.getActivePotionEffect(ModEffects.armor_regeneration);
+                double armorRegenerationMod = armorRegen == null ? 0 : armorRegen.getDuration() / ((double)VampirismConfig.BALANCE.vpNaturalArmorRegenDuration.get() * 20);
+                naturalArmor *= (1-0.5*armorRegenerationMod); //Modify natural armor between 50% and 100% depending on the armor regen state
```

Comment:
is 50% a little  too much? I would prefer a lower base value and a higher regeneration.

Reply:
I thought iron might be a good starting point (if you are at level 14). But yeah, it should regenerate fairly quickly, so I changed it to 25% for now